care/facility/api/serializers/asset.py

```diff
@@ -21,6 +22,14 @@
 
 
 class AssetLocationSerializer(ModelSerializer):
+    name = serializers.CharField(
+        validators=[
+            UniqueValidator(
+                queryset=AssetLocation.objects.all(),
```

Comment:
This is applied across all asset locations, not just in a given facility.

Reply:
Removed.